### PR TITLE
Doc edit

### PR DESCRIPTION
**Pull request by g-hodges**
The edits standardize comma usage and parallel list construction.

**Summary**

These changes improve comma placement and list consistency.

**Test plan**

NA

porch-time

### CODE REVIEW

**Comment by verythorough** [comment]
Deploy preview for *netlify-cms-www* ready!

Built with commit 9beb76e54c47f0072c001d196534e7e079f669e5

https://deploy-preview-1677--netlify-cms-www.netlify.com

**Comment by verythorough** [comment]
Deploy preview for *cms-demo* ready!

Built with commit 9beb76e54c47f0072c001d196534e7e079f669e5

https://deploy-preview-1677--cms-demo.netlify.com

File: website/content/docs/architecture.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ group: contributing
 
 Netlify CMS is a React application, using Redux for state management with immutable data structures (immutable.js).
 
-The core abstractions for content editing are `collections`, `entries` and `widgets`.
+The core abstractions for content editing are `collections`, `entries`, and `widgets`.
```
Comment (erquhart) [inline website/content/docs/architecture.md]: oxford comma forever 💪

**Comment by erquhart** [comment]
Beautiful edits, we will take any and all contributions from you my friend 👍

**Pull request merged**